src/modules/markets/MarketAssetsListContainer.tsx

```diff
@@ -109,31 +114,45 @@ export const MarketAssetsListContainer = () => {
       {frozenOrPausedReserves.length > 0 && (
         <Box sx={{ mt: 10, px: { xs: 4, xsm: 6 } }}>
           <Typography variant="h4" mb={4}>
-            <Trans>Frozen or paused assets</Trans>
+            <Trans>Show Frozen or paused assets</Trans>
+
+            <Switch
+              checked={showFrozenMarketsToggle}
+              onChange={handleChange}
+              inputProps={{ 'aria-label': 'controlled' }}
+            />
           </Typography>
-          <Warning severity="info">
-            <Trans>
-              These assets are temporarily frozen or paused by Aave community decisions, meaning
-              that further supply / borrow, or rate swap of these assets are unavailable.
-              Withdrawals and debt repayments are allowed. Follow the{' '}
-              <Link
-                onClick={() => {
-                  trackEvent(GENERAL.EXTERNAL_LINK, {
-                    link: 'Frozen Market Markets Page',
-                    frozenMarket: currentNetworkConfig.name,
-                  });
-                }}
-                href="https://governance.aave.com"
-                underline="always"
-              >
-                Aave governance forum
-              </Link>{' '}
-              for further updates.
-            </Trans>
-          </Warning>
+          {showFrozenMarketsToggle ? (
```

Comment:
Since the false case doesn't render anything, you could just use `&&` to simplify it:

```
{showFrozenMarketsToggle && (...)}
```

same for the one on line 151

Reply:
updated in fb61b0754500304406c74dcc0f747f9a4d38e937